web-ui/src/main/resources/catalog/views/default/templates/results.html

```diff
@@ -1,34 +1,34 @@
-<div class="container-fluid"
-     data-ng-search-form=""
+<div data-ng-search-form=""
      data-runSearch="true">
   <div data-ng-include="'../../catalog/views/default/templates/searchForm.html'"></div>
 
   <div gn-grid-related-query="searchResults.records"></div>
-  <div class="row gn-row-results">
-    <div class="col-md-3 gn-search-facet">
+  <div class="row gn-row-results" style="background-color:red;">
```

Comment:
Do we really want such a red background :)

![image](https://user-images.githubusercontent.com/1701393/56365987-62424180-61f2-11e9-884f-1d9ccfc154b4.png)

Reply:
I'm fixing that during merge